### DISCUSSION TITLE
Toggle login

### DISCUSSION THREAD

**Discussion by a3be3** [ppy/osu#19388] [Ideas] [answered]
Now, if we are offline, we should input username & password to login, and this is so annoying for me because I often use offline mode.
This is almost for tournament mappers (by the way I'm a member of EGTS custom mappers), but I really want you to add toggle (switching) login system.

**Comment by peppy** [comment, marked as answer]
![osu! 2022-07-26 at 05 06 05](https://user-images.githubusercontent.com/191335/180927634-38690b9e-c208-4a0c-864e-7e779dcbeb9d.png)

Note this doesn't quite work right now, but will be implemented when there's more use for it. Right now other users cannot see what you are currently doing in the editor, at least.

**Reply by a3be3** [reply]
In other words, are there any plans to implement that in the stable version (not razer), right?
In that case I'll wait for the razer version to be completed!

**Reply by peppy** [reply]
There are not, no.
